src/main/java/uk/ac/cam/cl/dtg/isaac/api/EventsFacade.java

```diff
@@ -1295,22 +1249,19 @@ public final Response resendEventEmail(@Context final HttpServletRequest request
 
     /**
      * Delete a booking.
-     *
+     * <p>
```

Comment:
Hmmn. I was playing with the settings. Technically newlines in JavaDoc are meaningless, since it is HTML-like and so you need `<p>` tags to break up sections. But we _don't_ use a JavaDoc site generator, and so have historically just treated the JavaDoc as slightly-complicated structured-comments.

IntelliJ automatically adds the `<p>` tags when you reformat a file, but I think I persuaded it to stop doing it and this one slipped through.

Reply:
Fair enough - as it stands we haven't done this in most places, but have in some. I only mention it in case it implies a problem with the new config. At any rate, not worth holding up the PR over!